bootstrap-salt.sh

```diff
@@ -4427,10 +4427,20 @@ install_amazon_linux_ami_deps() {
     _USEAWS=$BS_FALSE
     pkg_append="python"
 
-    repo_rev="$(echo "${STABLE_REV}"  | sed 's|.*\/||g')"
+    if [ "$ITYPE" = "stable" ]; then
+        repo_rev="$STABLE_REV"
+    else
+        repo_rev="latest"
+    fi
+
+    if [[ $repo_rev =~ ^archive ]]; then
```

Comment:
This will definitely work with `bash` version provided by Amazon Linux, but we aim to keep the bootstrap script fully written in POSIX shell. Please consider to use `egrep` for `$repo_rev` value validation. Just like at line 4442.